R-package/src/xgboost_R.cc

```diff
@@ -280,11 +296,15 @@ SEXP XGBoosterBoostOneIter_R(SEXP handle, SEXP dtrain, SEXP grad, SEXP hess) {
       << "gradient and hess must have same length";
   int len = length(grad);
   std::vector<float> tgrad(len), thess(len);
+  OMP_INIT();
```

Comment:
Could you please consider using `dmlc::OMPException` instead of macros?

Reply:
Done in 4fb1b75.